Content.Server/GameObjects/Components/Items/DiceComponent.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using Content.Server.GameObjects.EntitySystems;
+using Robust.Server.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Log;
+using Robust.Shared.Maths;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Items
+{
+    public class DiceComponent : Component, IActivate, IUse, ILand, IExamine
+    {
+        public override string Name => "Dice";
+
+        private readonly Random _random = new Random();
```

Comment:
You need to seed this random instance better. By default it uses the current time imprecisely, so if two dice get made at the same time (say, map load...) they'll get the same seed.

Initialize it in `OnAdd` with like `new Random(Owned.Uid.GetHashCode() ^ DateTime.Now.GetHashCode())`

Reply:
Done.